images/core.proto

```diff
@@ -106,10 +106,13 @@ message thread_core_entry {
 	optional seccomp_mode		seccomp_mode	= 11;
 	optional uint32			seccomp_filter	= 12;
 
-	optional string			comm		= 13;
-	optional uint64			blk_sigset_extended	= 14;
-	optional rseq_entry		rseq_entry	= 15;
-	optional uint32			cg_set		= 16;
```

Comment:
Changing the field numbers here would break backward compatibility: https://criu.org/Postulates

Reply:
Got it. I'll move the sud_mode and _setting to the bottom of the struct